src/gallery/slider.js

```diff
@@ -99,13 +100,16 @@ const clientWidth = window.innerWidth,
 			errorElement = container.querySelector( `.${prefixClassname}-item-loading-error` )
 
 		if ( refresh ) {
-			imageElement.src = `${imageElement.src}?timestamp=${Date.now()}`
```

Comment:
the timestamp is needed here because it tells the browser to retry the image, if no timestamp was given the browser will treat it like before state (possibly in cache or others)

Reply:
Hmm ok, how about including the `?timestamp` within `showImageInfo` when setting the image `src` there?

https://github.com/wikimedia/wikipedia-preview/pull/70/files#diff-d9d613b93342c10d9ee8b8ac7c9118f6R158

I updated this logic here to use `showImageInfo` instead of directly resetting the `src` because we need the api call to go through again when user is coming back from offline

Reply:
> I updated this logic here to use showImageInfo instead of directly resetting the src because we need the api call to go through again when user is coming back from offline

Make sense

> how about including the ?timestamp within showImageInfo when setting the image src there

It should work, but that means the image never **gets cached by the browser**, reload the image every time when the user comes back to the gallery slider view again.

Probably another parameters `showImageAndInfo( index, refreshImage = false)` and add timestamp when `refreshImage ` is true?